spec/lib/retriable_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+RSpec.describe Retriable do
+  subject(:call) {}
+
+  describe('#retries') do
+    let(:cat) { double }
+
+    it 'makes the call' do
+      allow(cat).to receive(:maow)
```

Comment:
TIL there are different spellings for cat vocalisations :tada: https://en.wikipedia.org/wiki/Meow

Reply:
How do you feel about making our cat a kitten and having vocalise as `mew` :laughing:

Reply:
OMG you are right! Maow is actually a Canadian rock band: https://en.wikipedia.org/wiki/Maow